Ruby Basic/task2.rb

```diff
@@ -0,0 +1,54 @@
+class String
+  
+  def black
+    "\e[30m#{self}\e[0m"
+  end
+
+  def red
+    "\e[31m#{self}\e[0m"
+  end
+
+  def green
+    "\e[32m#{self}\e[0m"
+  end
+
+  def brown
+    "\e[33m#{self}\e[0m"
+  end
+
+  def blue
+    "\e[34m#{self}\e[0m"
+  end
+
+  def magenta
+    "\e[35m#{self}\e[0m"
+  end
+
+  def cyan
+    "\e[36m#{self}\e[0m"
+  end
+
+  def gray
+    "\e[37m#{self}\e[0m"
+  end
+end
+
+color_palette = ['black', 'red', 'green', 'gray', 'blue', 'brown', 'magenta', 'cyan']
+
+input_data = ARGV
```

Comment:
it is useless variable, remove it please

Reply:
Done